content/english/phaser-space-invaders-game/activity-2.md

```diff
@@ -7,13 +7,17 @@ weight: 8
 
 To add some life to our game, let’s get things moving around! Animate the player we have created.
 
-![animation_part](../media/4/animation_part.png)
+<!--- ![animation_part](../media/4/animation_part.png) --->
+<img src="../media/4/animation_part.png" alt="animation_part" style="width:950px;"/>
 
-![play_animation](../media/4/play_animation.png)
+<!--- ![play_animation](../media/4/play_animation.png) --->
+<img src="../media/4/play_animation.png" alt="play_animation" style="width:950px;"/>
 
 Once your ship is animated, it should look like this. Notice the flames coming out of the ship's engine! (although it may be faster or slower depending on the frame rate that you picked)
```

Comment:
```suggestion
Once your ship is animated, it should look like this. Notice the flames coming out of the ship's engine! (Although it may be faster or slower depending on the frame rate that you picked).
```